lib/app.js

```diff
@@ -1,18 +1,27 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
+const cors = require('cors');
 
-const authenticate = require('./middleware/authenticate');
+// const authenticate = require('./middleware/authenticate');
```

Comment:
you can just remove this -- it was included in this file so you can add it to the items controller since all of those routes require you to be logged in -- saves you adding it to each individual route.